docs/can-guides/experiment/technology/technology.md

```diff
@@ -0,0 +1,600 @@
+@page guides/technology-overview Technology Overview
+@parent guides/getting-started 1
+@outline 3
+
+@description Learn the basics of the core parts of CanJS's technology.
+
+@body
+
+## Overview
+
+CanJS, at it's most simplified, consists of key-value __Observables__
+connected to different web browser APIs through various connecting
+libraries.
+
+<img src="../../docs/can-guides/experiment/technology/overview.png"
+  alt="Observables are the center hub.  They are connected to the DOM by the view layer, the service layer by the data modeling layer, and the window location by the routing layer"
+  width="800px"/>
+
+The general idea is that you create observable objects that encapsulate
+the logic and state of your application and connect those observable
+objects to:
+
+- The Document Object Model (DOM) to update your page automatically.
+- The route to support the forward and back button.
+- Your service layer to make receiving, creating, updating, and deleting server data easier.
+
+
+Instead of worrying about calling the various browser APIs, CanJS abstracts this away, so you can focus on the logic of your application. The logic of your application is contained within observables. Lets see how!
+
+
+## Key-Value Observables
+
+The [can-define/map/map DefineMap] and
+[can-define/list/list DefineList] __Observables__ define the logic and state
+in your application. For example, if we wanted to model
+a simple counter, we can use [can-define/map/map DefineMap] as follows:
+
+```js
+import DefineMap from "can-define/map/map";
+var Counter = DefineMap.extend("Counter",{
```

Comment:
The mixture of `import` and `var` stand out; any reason not to use `const` and `let` in this document?

Reply:
`const` and `let` are for the weak minded.  I'm part of the hard-core`var` uproar!